test/cookbooks/python-webapp-test/templates/default/config.yml.erb

```diff
@@ -0,0 +1,14 @@
+DEBUG: <%= @debug %>
+
+item1: <%= @item1 %>
+item2: <%= @item2 %>
+
+##### Database Configuration #####
+DATABASES:
+    default:
+        ENGINE: "<%= @database[:settings][:adapter] %>"
+        NAME: "<%= @database[:settings][:name] %>"
+        USER: "<%= @database[:settings][:username] %>"
+        PASSWORD: "<%= @database[:settings][:password] %>"
+        HOST: "<%= @database[:settings][:host] %>"
+        PORT: <%= @database[:settings][:port]  %>
```

Comment:
This is for the Seagrant Twins' bizarro yaml configuration system:
https://github.com/osu-cass/whats-fresh-api/blob/develop/whats_fresh/settings.py